packages/framework-integration-tests/integration/providers/aws/utils.ts

```diff
@@ -248,6 +248,38 @@ export async function queryEvents(primaryKey: string, latestFirst = true): Promi
   return output.Items
 }
 
+// --- Read models helpers ---
+
+export async function readModelTableName(readModelName: string): Promise<string> {
+  const stackName = appStackName()
+
+  return `${stackName}-${readModelName}`
+}
+
+export async function queryReadModels(primaryKey: string, readModelName: string, latestFirst = true): Promise<any> {
+  const output: QueryOutput = await documentClient
+    .query({
+      TableName: await readModelTableName(readModelName),
+      KeyConditionExpression: 'id = :v',
```

Comment:
i would give `:v` a more meaningful name. I understand that it means `value` but since this expression will be always targeting the primaryKey, what about having something like `:primaryKey`?